File: C_Programming/Day_1/program2.c
// Description : Write a program to calculate the percentage .

#include<stdio.h>

int CalculatePercentage(int iObtained , int iTotal)
{
    float fPercentage = 0.0f;

    fPercentage = (float)iObtained / (float)iTotal * 100;
    
    return fPercentage;
}

int main()
{
    int iObtained = 0;
    int iTotal = 0;
    float fPercentage = 0.0f;

    printf("Enter the obtained marks : ");
    scanf("%d",&iObtained);

    printf("Enter the total marks : ");
    scanf("%d",&iTotal);

    fPercentage = CalculatePercentage(iObtained , iTotal);

    printf("The percentage is : %f",fPercentage);

    return 0;
}

File: Python_Programming/Day_1/program2.py
# Description : Write a program to calculate the percentage .

def CalculatePercentage(Obtained , Total):
    Percentage = 0.0

    Percentage = Obtained/Total *100

    return Percentage


def main():
    Obtained = int(input("Enter your obtained marks : "))
    Total = int(input("Enter your total marks : "))

    Percentage = CalculatePercentage(Obtained , Total)

    print("Total percentage is : ",Percentage)

if __name__ == "__main__":
    main()